spec/models/vulnerability_template_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe VulnerabilityTemplate, type: :model do
+  let(:identifier) { SecureRandom.uuid }
+  let!(:vulnerability_template) { described_class.new(identifier: identifier) }
+
+  describe 'uri' do
+    it 'only accepts a valid uri' do
+      vulnerability_template.uri = 'lammajama_232'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.